2020-an-environment-for-sustainable-research-software-in-germany-and-beyond/manuscript.md

```diff
@@ -0,0 +1,2039 @@
+38. Institute of Nanotechnology, Karlsruhe Institute of Technology (KIT), Germany and 
+39. Institute of Applied Mathematics, Leibniz University Hannover, Germany and 
+40. Energy Technology, Eindhoven University of Technology, The Netherlands and 
+41. Julius Kühn-Institut (JKI), Federal Research Centre for Cultivated Plants, Quedlinburg, Germany and
+42. LMU Munich, Germany and
+43. Bielefeld University, Germany and
+44. Helmholtz Zentrum München, Germany
+45. Max Planck Institue of Molecular Cell Biology and Genetics, Dresden, Germany
+46. Deutsches Krebsforschungszentrum, Heidelberg, Germany
+47. Chair of Scientific Computing in Computer Science, Technical University Munich, Germany
+- * axel.loewe@kit.edu, gunnar.seemann@universitaets-herzzentrum.de
+- † Contributed equally
+bibliography: 'references.bib'
+title: |
+    An Environment for Sustainable Research Software in Germany and Beyond:
+    Current State, Open Challenges, and Call for Action
+---
+
+Sustainable Software Development; Academic Software; Software
+Infrastructure; Software Training; Software Licensing; Research Software
+
+Background
+==========
+
+> Meet Kim, who is currently a post-grad PhD student in researchonomy at
+> the University of Arcadia. We will follow Kim’s fictional career in
+> order to understand different aspects of research software
+> sustainability. Note that in Kim’s world, many of the changes this
+> paper calls for have already been implemented.  
+> (In our example, Kim is a female person. Of course, research software
+> engineers (RSEs) can be any gender.)
+
+Computational analysis of huge data sets, computer-based simulations,
+and software technology play a central role for virtually all scientific
+breakthroughs of the 21st century. The first “picture” of a black hole
+may be the most prominent recent example where astrophysical experiments
+and the collection and processing of data had to be complemented with
+sophisticated algorithms and software to enable research excellence (The
+Event Horizon Telescope Collaboration et al. 2019). Similarly, it is
+research software that allows us to get a glimpse of the consequences
+our actions today have on the climate of tomorrow. However, an
+implication of computer-based research is that findings and data can
+only be reproduced, understood, and validated if the software that was
+used in the research process is available and
+maintained.  
+At the same time, sustaining research software, and in particular open
+research software, comes with a number of challenges. Commercial
+research software often has revenue flows that can facilitate
+sustainable software development, maintenance, and documentation as well
+as the operation of adequate infrastructure. A large share of researchers (want to or have to) 
+base their research on software that is often developed in-house or as a community
+effort. Many of these software stacks can not be
```

Comment:
```suggestion
effort. This can be due to a newly developed algorithm
and the resulting software being at least part of the research work itself,
or due to a highly dynamic field of research with diverse research questions
requiring specialized and quickly adapting software solutions that do not
offer sufficient financial incentives for commercial software solutions to
be developed, or due to disadvantages of commercial software outweighting
its advantages in the specific use case (e.g. prohibitive license costs,
proprietary not publically documented algorithms or export restrictions).
This need of custom in-house or community software development notwithstanding,
the approach brings its own significant challenges: Many of these software stacks can not be
```

From #6